scalafmt-tests/src/test/resources/newlines/source_classic.stat

```diff
@@ -3791,3 +3791,314 @@ type T = A1 & A2 | A3 & A4 | A5 &
   A64 | A65 &
   A66 /* c1 */ | /* c2 */ A67 & A68 |
   A69 & A70[T70]
+<<< #2561 scala 2: defn, no align
+align.preset = none
+newlines.beforeOpenParenDefnSite = true
+newlines.sometimesBeforeColonInMethodReturnType = true
+===
+object a {
+  def fooFunc
+    (foo1: String)
+    (foo2: String)
+    : String = ???
+  def fooFunc
+    (foo1: String)
+    (foo2: String)
+    (foo3: String)
+    (foo4: String)
+    (foo5: String)
+    : (String, String, String, String) = ???
+  def notAnInterestingFunction
+    (foo1: String)
+    (foo2: String, foo3: String, foo4: Map[String, Map[String, Map[String, String]]]) // this line doesn't fit
+    (foo5: String)
+    (implicit foo6: Int,  foo7: Int,  foo8: Int,  foo9: Int,  foo10: Int,  foo11: Int,  foo12: Int) // this line also doesn't fit
+    : (String, String, String, String) = ???
+}
+>>>
+object a {
+  def fooFunc(foo1: String)
+    (foo2: String)
+    : String = ???
```

Comment:
@tgodzik @poslegm @djneades was debating whether to indent here... opinions?

Reply:
@kitbellew I’m not sure I quite understand the question. May I ask what the alternatives would be?

Reply:
two options:
```
def fooFunc(foo1: String)
  (foo2: String)
  : String = ???
```
or
```
def fooFunc(foo1: String)
  (foo2: String)
: String = ???
```

Reply:
Ah, thank you! The former would definitely be my preference.

Note that, with the function body not fitting on the same line as the type signature there, I’d actually want to double-indent the argument blocks and return type, like this:

```scala
def fooFunc
    (foo1: String)
    (foo2: String)
    : String =
  ???
```

Reply:
increase the `indent.defnSite`.

Reply:
Thank you. I do have that in my current `.scalafmt.conf`; I just wanted to be sure it would still work with the new layout options.